profile/CODE_OF_CONDUCT.md

```diff
@@ -61,7 +61,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+scipp.ess.eu.
```

Comment:
```suggestion
scipp[at]ess.eu.
```
to show that this is an email address. We used @ in the scipp repo. `[at]` probably doesn't provide much protection nowadays but I'd still prefer it.